src/test/groovy/com/adarshr/gradle/testlogger/logger/ConsoleLoggerSpec.groovy

```diff
@@ -40,4 +40,14 @@ class ConsoleLoggerSpec extends Specification {
         then:
             1 * loggerMock.log(LIFECYCLE, '')
     }
+
+    def "configurable log level"() {
+        given:
+            consoleLogger = new ConsoleLogger(loggerMock, DEBUG, textRendererMock)
+            textRendererMock.render('text to be logged') >> 'rendered ansi text'
+        when:
+        consoleLogger.log('text to be logged')
+        then:
+        1 * loggerMock.log(DEBUG, 'rendered ansi text')
```

Comment:
Lines 49 and 51 should be indented according to the convention used for Spock tests in the rest of this project.